src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs

```diff
@@ -100,7 +101,7 @@ internal IEnumerable<BluetoothGattService> GetServices(BluetoothGattServer serve
                 return true;
             };
 
-            int err = Interop.Bluetooth.BtGattServerForeachServices(_handle, cb, IntPtr.Zero);
+            int err = Interop.Bluetooth.BtGattServerForeachServices(_handle, _serviceForeachCallback, IntPtr.Zero);
```

Comment:
`BtGattServerForeachServices` is not async method, so need not change.
anyway it also no matter

Reply:
Thank you for comment. I want to leave it as it is.